gamestonk_terminal/cryptocurrency/coingecko/pycoingecko_view.py

```diff
@@ -0,0 +1,1570 @@
+        if not ns_parser:
+            return
+
+        df = gecko.get_stable_coins()
+        print(
+            tabulate(
+                df,
+                headers=df.columns,
+                floatfmt=".0f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def yield_farms(other_args: List[str]):
+    """Shows Top Yield Farming Pools by Value Locked from "https://www.coingecko.com/en/yield-farming"
+
+    Parameters
+    ----------
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="stablecoins",
+        add_help=False,
+        description="Shows Top Yield Farming Pools by Value Locked",
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = gecko.get_yield_farms()
+        print(
+            tabulate(
+                df,
+                headers=df.columns,
+                floatfmt=".0f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def top_volume_coins(other_args: List[str]):
+    """Shows Top 100 Coins by Trading Volume from "https://www.coingecko.com/en/yield-farming"
+
+    Parameters
+    ----------
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="top_volume",
+        add_help=False,
+        description="Shows Top 100 Coins by Trading Volume",
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = gecko.get_top_volume_coins()
+        print(
+            tabulate(
+                df,
+                headers=df.columns,
+                floatfmt=".2f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def top_defi_coins(other_args: List[str]):
+    """Shows Top 100 DeFi Coins by Market Capitalization from "https://www.coingecko.com/en/defi"
+    DeFi or Decentralized Finance refers to financial services that are built
+    on top of distributed networks with no central intermediaries.
+
+    Parameters
+    ----------
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="top_defi",
+        add_help=False,
+        description="Shows Top 100 DeFi Coins by Market Capitalization",
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = gecko.get_top_defi_coins()
```

Comment:
Add flag with number to show, by default I would just add top 10/20. The idea is that the user can get data fast and move fast. So, whenever possible we want the user to be able to reach the data he wants to see with the minimum overhead.

In this case I think is very unlikely that someone care about the coins that are in top 19 or 56. However, if it is in top 10 is probably meaningful.